src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString.split(';');
+  const result = {};
+
+  properties.forEach((element) => {
+    const [property, value] = element.split(':');
+
+    if (property && value) {
+      result[property.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. For better readability, you could chain the `split` and `forEach` with proper indentation.